### ISSUE TITLE
Microsoft.CSharp.RuntimeBinder.RuntimeBinderException : Cannot perform runtime binding on a null reference

### ISSUE THREAD

**Issue by wwExpert** [SciSharp/Keras.NET#10]
Hi,

I got this error message:

Microsoft.CSharp.RuntimeBinder.RuntimeBinderException : Cannot perform runtime binding on a null reference

I tried the xor example and also my own. I reinstalled Tensorflow with Pip, but it was not successful.

...

4/4 [==============================] - 0s 500us/step - loss: 0.0637 - acc: 1.0000
Epoch 1000/1000

2/4 [==============>...............] - ETA: 0s - loss: 0.0837 - acc: 1.0000
4/4 [==============================] - 0s 500us/step - loss: 0.0637 - acc: 1.0000
Ausnahme ausgelöst: "Microsoft.CSharp.RuntimeBinder.RuntimeBinderException" in System.Core.dll
Ein Ausnahmefehler des Typs "Microsoft.CSharp.RuntimeBinder.RuntimeBinderException" ist in System.Core.dll aufgetreten.
Die Laufzeitbindung kann für einen NULL-Verweis nicht ausgeführt werden.

**Comment by wwExpert** [comment]
More details of the error:

code:  model.Fit(x, y, batch_size: 1, epochs: 100, verbose: 1);

icrosoft.CSharp.RuntimeBinder.RuntimeBinderException: Cannot perform runtime binding on a null reference
   at CallSite.Target(Closure , CallSite , Object )
   at System.Dynamic.UpdateDelegates.UpdateAndExecute1[T0,TRet](CallSite site, T0 arg0)
   at Keras.Callbacks.Callback..ctor()
   at Keras.Models.BaseModel.Fit(NDarray x, NDarray y, Nullable`1 batch_size, Int32 epochs, Int32 verbose, Callback[] callbacks, Single validation_split, NDarray[] validation_data, Boolean shuffle, Dictionary`2 class_weight, NDarray sample_weight, Int32 initial_epoch, Nullable`1 steps_per_epoch, Nullable`1 validation_steps)
   at testKereas.Program.Main(String[] args) in

**Comment by deepakkumar1984** [comment]
Can you please share the code?

**Comment by Zuntara** [comment]
I have the same issue in number 8 - also on the callback but it's in the model.add function, in another example it has the same issue with model.fit

**Comment by wwExpert** [comment]
Hi, this is the whole code of the program.

using System;
using Keras;
using Keras.Layers;
using Keras.Models;
using Numpy;
using Python;
using System.IO;


namespace testKereas
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello Keras XOR!");

                   

            //var board =  Keras.Callbacks.TensorBoard(log_dir: "./logs", histogram_freq: 0, batch_size: 32, write_graph: true, write_grads: false, write_images: false, embeddings_freq: 0, embeddings_layer_names: null, embeddings_metadata: null, embeddings_data: null, update_freq: "epoch");

            // Load train data
            NDarray x = np.array(new float[,] { { 0, 0 }, { 0, 1 }, { 1, 0 }, { 1, 1 } });
            NDarray y = np.array(new float[] { 0, 1, 1, 0 });

            //Build sequential model
            var model = new Sequential();
            model.Add(new Dense(32, activation: "relu", input_shape: new Shape(2)));
            model.Add(new Dense(64, activation: "relu"));
            model.Add(new Dense(1, activation: "sigmoid"));

            //Compile and train
            model.Compile(optimizer: "sgd", loss: "binary_crossentropy", metrics: new string[] { "accuracy" });
            try
            {
                var hist = model.Fit(x, y, batch_size: 1, epochs: 100,1,null,0,null,true,null,null,0,null,null);
            }
            catch(Exception ex)
            {
                Console.WriteLine(ex.ToString());
            }

            //Save model and weights
            string json = model.ToJson();
            File.WriteAllText("model.json", json);
            model.SaveWeight("model.h5");

            //Load model and weight
            var loaded_model = Sequential.ModelFromJson(File.ReadAllText("model.json"));
            loaded_model.LoadWeight("model.h5");




            Console.ReadLine();

**Comment by wwExpert** [comment]
Hi, with the last update of Keras.Net it works for me. Thank´s a lot for your work!
